install_plugin.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+
+export VAULT_ADDR='http://127.0.0.1:8200'
+
+vault login root
```

Comment:
Rely on $GOPATH var
I would put the vault binary in  `$GOPATH/bin/` and align here

Reply:
Also, parametrize the host and uri